tests/integration/handling-api-messages.js

```diff
@@ -0,0 +1,20 @@
+describe("handling api messages", function () {
+
+  it("handles and re-hydrates commands", function () {
```

Comment:
Can you point me to the part where the command is 'dehydrated'? I'm a little lost following this though :-)